tests/test_cmd_autoupdate.py

```diff
@@ -99,9 +99,9 @@ def test_autoupdate_multiple_workflows(self):
                 with open(wf_file) as g:
                     wf = json.load(g)
                 # check tool within parent wf has updated
-                assert wf["steps"]["1"]["tool_version"] == "3.7+galaxy0"
+                assert wf["steps"]["1"]["tool_version"] > "3.6+galaxy1"
```

Comment:
That would compare in alphabetic order. I think it's fine to just assert it's not `3.6+galaxy1`.